src/dynamics/initial_conditions.jl

```diff
@@ -302,6 +302,10 @@ Parameters are $(TYPEDFIELDS)"""
     m::Int = 4
     ω::Float64 = 7.848e-6
     K::Float64 = 7.848e-6
+    Ω::Float64 = 7.292e-5
```

Comment:
you find this in `model.planet.rotation`, this way it's consistent with the rotation rate as defined in the planet